google-login-plugin/src/com/google/gct/login/stats/GctConfigurableProvider.java

```diff
@@ -0,0 +1,30 @@
+package com.google.gct.login.stats;
+
+import com.intellij.openapi.options.Configurable;
+import com.intellij.openapi.options.ConfigurableProvider;
+import com.intellij.util.PlatformUtils;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Registers an implementation of {@code applicationConfigurable} extension to provide a
+ * Google Cloud Tools tab in the "Settings" dialog if current application is IntelliJ.
+ */
+public class GctConfigurableProvider extends ConfigurableProvider {
```

Comment:
GoogleSettingsConfigurableProvider

I wonder if we should be rebranding Google Login plugin as something more like Google Account Plugin or something like that.  A basis for common behavior needed for all Google plugins.